minikube-gradle-plugin/src/main/java/com/google/cloud/tools/minikube/util/CommandExecutor.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.minikube.util;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+import org.gradle.api.GradleException;
+import org.gradle.api.logging.Logger;
+import org.gradle.internal.impldep.com.google.common.annotations.VisibleForTesting;
+
+/** Executes a shell command. */
+public class CommandExecutor {
+  public CommandExecutor() {}
+
+  public CommandExecutor setLogger(Logger logger) {
+    this.logger = logger;
+    return this;
+  }
+
+  @VisibleForTesting static final int TIMEOUT_SECONDS = 5;
+
+  @VisibleForTesting
```

Comment:
Don't use this annotation from `org.gradle.internal.impldep.com.google.common.annotations.VisibleForTesting`, we don't pull in the dependency ourselves to guava, so just either leave it in but commented, or take it out.